Tests/ObfuscationTests.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  ObfuscationTests.swift
+//  UtiliKit-iOSTests
+//
+//  Created by Russell Mirabelli on 7/23/19.
+//  Copyright © 2019 CocoaPods. All rights reserved.
+//
+
+import XCTest
+@testable import UtiliKit
```

Comment:
I'm thinking a regular `import` might work just as well here since we're testing public APIs (also, we might need to make much `ObfuscatedKey.swift` `public` so that it's usable in client code).

Reply:
done